ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/footer/v1/footer/README.md

```diff
@@ -0,0 +1,38 @@
+<!--
+Copyright 2022 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Footer (v1)
+====
+Footer component written in HTL.
+
+## Features
+
+* Configurable text in footer
+* Configurable HTML ID attribute.
+
+
+### Use Object
+The Panel Container component uses the `/libs/core/wcm/components/text/v2/text` component and its Sling model as its Use-object.
+
```

Comment:
BEM `cmp-adaptiveform-footer__text`  shoujld also be mentioned.

Reply:
yeah correct, have made the change